lib/ldap.c

```diff
@@ -464,6 +464,11 @@ static CURLcode ldap_do(struct Curl_easy *data, bool *done)
 #endif
 #endif /* CURL_LDAP_USE_SSL */
   }
+  else if(data->set.use_ssl > CURLUSESSL_TRY) {
+    failf(data, "LDAP local: explicit TLS not supported");
+    result = CURLE_USE_SSL_FAILED;
```

Comment:
Maybe `CURLE_NOT_BUILT_IN` ? Because this happens when the user selects an option that *could* be supported and work if libcurl was just built differently.

Reply:
Makes sense.
My primary idea was to emphasize on an SSL problem and avoid the same error as if LDAP was not compiled in.